test/delete_cr/deletecr_test.go

```diff
@@ -54,7 +54,8 @@ var _ = Describe("DeleteCR test", func() {
 	Context("Multisite cluster deployment (M13 - Multisite indexer cluster, Search head cluster)", func() {
 		It("deletecr: can deploy indexers and search head cluster", func() {
 
-			err := deployment.DeploySingleSiteCluster(deployment.GetName(), 3)
+			shc := true
```

Comment:
Nit: Instead of using a variable here. You could directly call the function as follows to indicate that the bool is to indicated whether you want a SHC:

`err := deployment.DeploySingleSiteCluster(deployment.GetName(), 3, /*shc*/true)`